### ISSUE TITLE
add support "percentage" value for "column-gap"

### ISSUE THREAD

**Issue by horans** [CSSLint/csslint#745]
when using a `percentage` value for `column-gap`, i received the `known-properties` warning.

```css
dl { column-count: 2; column-gap: 18%; } /* csslint allow: known-properties */
```

references:
https://drafts.csswg.org/css-multicol-1/#cg
https://developer.mozilla.org/en-US/docs/Web/CSS/column-gap
